<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmBuscarFactura</class>
 <widget class="QDialog" name="FrmBuscarFactura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscar Facturas</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Buscar Facturas entre:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="FechaInicio">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="FechaFin">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Cuyo importe esté entre: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Importeminimo">
       <property name="text">
        <string>0,00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ImporteMaximo">
       <property name="text">
        <string>99.999.999,00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ver sólo facturas de:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="txtBuscar"/>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Buscar</string>
       </property>
       <property name="icon">
        <iconset resource="../terra.qrc">
         <normaloff>:/Icons/PNG/search.png</normaloff>:/Icons/PNG/search.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QTableView" name="Grid"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="styleSheet">
        <string notr="true">QPushButton:disabled{color: 0; }
QPushButton:enabled{color: rgb(255,255,255); }</string>
       </property>
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="styleSheet">
        <string notr="true">QPushButton:disabled{color: 0; }
QPushButton:enabled{color: rgb(255,255,255); }</string>
       </property>
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtBuscar</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Grid</tabstop>
 </tabstops>
 <resources>
  <include location="../terra.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>FrmBuscarFactura</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAceptar</sender>
   <signal>clicked()</signal>
   <receiver>FrmBuscarFactura</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
